site/src/components/message.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - message
+---
+
+# Message
+
+Use message component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of message component.
+
+```html
+<article class="message">
+	<p class="message-time">May 29, 2023, 4:50PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="margin-bottom-xxs">
+			<div class="message-content-header">
+				<p class="message-content-display-name">Q&A Moderator</p>
+				<div class="message-content-options">
+					<span class="icon primary-dark">
```

Comment:
1. primary-dark isn't a class
2. maybe we just paste in `⋮`?

Reply:
updated